src/style.css

```diff
@@ -0,0 +1,120 @@
+html {
```

Comment:
please create separate css files for each BEM block
![image](https://user-images.githubusercontent.com/22824431/227714627-932170f1-fb79-4f2a-be14-683068feec43.png)